.github/auto-assign.yml

```diff
@@ -0,0 +1,22 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to author to set pr creator as assignee
+addAssignees: author
+
+# A list of reviewers to be added to pull requests (GitHub user name)
+reviewers:
+  - eyusufatik
+  - kpp
+  - rakanalh
+  - yaziciahmet
+  - jfldde
+  - ercecan
+
+# A list of keywords to be skipped the process that add reviewers if pull requests include it
+skipKeywords:
+  - wip
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 0
```

Comment:
In this case it would randomly select 2. But since @eyusufatik wants to review and approve all PRs then it might as well request the whole team for review and we can fine-tune later for how that works.

Reply:
Another alternative is to:
1. Specify @eyusufatik as a codeowner for the whole repo.
2. Set 2 reviewers to be randomly chosen excluding Esad.

@eyusufatik wdyt?